static/docs/user-guide/introduction.md

```diff
@@ -0,0 +1,31 @@
+Introduction
+------------
```

Comment:
should be # instead of `________`

Reply:
we probably need an introduction as well, but the issue 53 is more about introducing basic concepts. In this case I would name this page `Basic Concepts`.